fe/fe-core/src/main/java/com/starrocks/scheduler/persist/RunningTaskRunProgressInfo.java

```diff
@@ -0,0 +1,43 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.scheduler.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Map;
+
+
+public class RunningTaskRunProgressInfo implements Writable {
```

Comment:
TaskRunStateUpdateInfo may be better name?

Reply:
There is already a TaskRunStatusChange class , will it be ambiguous？